recipe/meta.yaml

```
@@ -0,0 +1,181 @@
+{% set name = 'hail' %}
+{% set version = '0.2.127' %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  # the build process requires a repository to set some variables
+  # (e.g. version)
+  git_url: https://github.com/hail-is/hail.git
+  git_rev: {{ version }}
+  patches:
+    - patches/0001-remove-vendored-libsimdpp.patch
+    - patches/0002-remove-vendored-catch2.patch
+    - patches/0003-patch-requirements.patch
+    - patches/0004-pluggy-old-style-wrappers.patch
+    - patches/0005-patch-spark-3.3.2-to-3.4.1.patch
+
+build:
+  number: 0
+  # avro and others a not available for win
+  # openjdk and pyspark not available for s390x
+  skip: true  # [py<39 or win or py>311 or s390x]
+  skip: true  # [blas_impl == "mkl"]
+  entry_points:
+    - hailctl = hailtop.hailctl.__main__:main
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - make      # [linux]
+    - patch     # [not win]
+    - m2-patch  # [win]
+    - git
+  host:
+    - python
+    - pip
+    - setuptools
+    # prerequisites
+    - pytest-runner
+    - pyspark >=3.3.2,<=3.4.1
+    - openjdk 11.*
+    - lz4
+    # blas (lapack is brought in along)
+    - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
```

```suggestion
```
openblas-devel has a run export for libopenblas, which is what you actually need at runtime
```suggestion
    - openblas-devel {{ openblas }}  # [blas_impl == "openblas"]
```